db/migrate/20231016233836_add_full_name_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddFullNameToUser < ActiveRecord::Migration[6.1]
```

Comment:
Why do we have this migration? we can have this into the same migration => AddFieldToUser